methods/list.js

```diff
@@ -6,6 +6,8 @@ module.exports = function(callback) {
   const s3Options = {};
   s3Options.Bucket = options.s3Bucket;
   s3.listObjectsV2(s3Options, (err, data) => {
-    callback(err, err ? undefined : data.Contents);
+    callback(err, err ? undefined : data.Contents.map((imageOnS3) => {
```

Comment:
just an fyi, this won't pass eslint. make sure you have it installed in your editor.  also it's not very readable